clang/docs/ReleaseNotes.rst

```diff
@@ -870,9 +870,9 @@ Bug Fixes to AST Handling
 - Fixed a bug where RecursiveASTVisitor fails to visit the
   initializer of a bitfield.
   `Issue 64916 <https://github.com/llvm/llvm-project/issues/64916>`_
-- Fixed a bug where Template Instantiation failed to handle Lambda Expressions
-  with certain types of Attributes.
-  (`#76521 <https://github.com/llvm/llvm-project/issues/76521>`_)
```

Comment:
Why did you remove that?

Reply:
in my previous mistake I think it got removed unintentionally ,sorry from my end.

Reply:
Should I add that to a new commit?

Reply:
Yes, please

Reply:
corrected that.